dind/Makefile

```diff
@@ -19,3 +19,13 @@ docker-tag:
 docker-push:
 	docker push $(IMAGE_NAME):$(IMAGE_VERSION)$(IMAGE_VARIANT)-$(DOCKER_VERSION)
 	docker push $(IMAGE_NAME):$(IMAGE_VERSION)$(IMAGE_VARIANT)
+
+docker-save:
+	docker save $(IMAGE_NAME):$(IMAGE_VERSION)$(IMAGE_VARIANT)-$(DOCKER_VERSION) > ./image.tar
+
+docker-load:
+	docker load < ./image.tar
+
+continuous-build: docker-build docker-save
+
+continuous-push: docker-load docker-tag docker-push
```

Comment:
Is it standard on our repos to have CI-related operation inside the Makefiles? I'd rather have the Makefile as clean and simple as possible, mostly oriented towards developer actions, than have the CI config file as simple and streamline as possible at the expense of the Makefile.

For exemple, if we're to use only CI/CD to deploy, I'd move all docker-related operation to the CI config file, except for operations we'd eventually need locally (e.g. build).

Reply:
This is our current internal recommendation for CI/CD YAMLs, describe the steps as much as possible inside the Makefile/Dockerfile and just write the necessary in the platform-specific YAML. This way we can migrate between CI/CD tools more easily without having to rewrite a bunch of non-standard configuration files.